packages/jobs/lib/execution/sync.ts

```diff
@@ -309,11 +342,14 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
                                 deleted
                             },
                             operation: lastSyncDate ? SyncType.INCREMENTAL : SyncType.FULL,
-                            logCtx
+                            logCtx: webhookLogCtx
                         });
 
                         if (res.isErr()) {
+                            await webhookLogCtx.failed();
                             throw new Error(`Failed to send webhook for sync: ${nangoProps.syncConfig.sync_name}`);
+                        } else {
+                            await webhookLogCtx.success();
```

Comment:
do you think it could be self-contained inside sendSyncWebhook?

Reply:
Definitely - we'd just need to send a handful more parameters to `sendSyncWebhook` to help it have all the log context params it needs. I'm not sure if that's ideal with the package separation, but we've already got a function called `sendSync` so it's not like the webhook package doesn't know about syncs.

Reply:
@bodinsamuel See ee9c652 for version where the log context is created inside of sendSync.